### DISCUSSION TITLE
clang-15: error: no such file or directory: '/std:c++17'

### DISCUSSION THREAD

**Discussion by CNSleepybear** [BoomingTech/Piccolo#437] [Q&A] [answered]
![C__Windows_system32_cmd exe 2023_3_5 21_53_44](https://user-images.githubusercontent.com/42902599/222964775-ceb866e9-4f88-46a3-a3e0-300e3f7d33d0.png)

请教下各位大佬，build引擎的时候报了这个错误，该怎么解决呢？

**Comment by ShenMian** [comment, marked as answer]
https://github.com/BoomingTech/Piccolo/blob/65379de9f67fe9c1d809e925c459d397e680ea8a/engine/3rdparty/JoltPhysics/Build/CMakeLists.txt#L29

JoltPhysics 中的 CMakeListst.txt 看上去是通过操作系统来判断可能的编译器类型, 所以在 Windows 下使用 clang 编译依然会转递给编译器 MSVC 的参数.  
可以尝试用 MSVC 进行编译, 或对相应的 CMakeLists.txt 文件进行修改.
